src/main/webapp/style.css

```diff
@@ -0,0 +1,47 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  size:80%;
+  
+}
+h1 {
+  background-color: #00a6a6;
+  padding: 5%;
+  text-align: center;
+  font-family: Helvetica;
+  color: white; 
+  font-size:1000%;
```

Comment:
Alternatively: font-size: 10em;

ems are a multiplier of the parent's size. Since the base font size is 16px (usually), this will be 160px most of the time. You can also use an rem (ex. font-size: 10rem) to base it always off the root element (<html> tag), so even if the parent isn't 16px, it'll still read from the base element and be 16px.